MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -34,8 +34,16 @@ public EncodingHelper(IMediaEncoder mediaEncoder, IFileSystem fileSystem, ISubti
 
         public string GetH264Encoder(EncodingJobInfo state, EncodingOptions encodingOptions)
         {
-            var defaultEncoder = "libx264";
+            return GetH264OrH265Encoder("libx264", "h264", state, encodingOptions);
+        }
 
+        public string GetH265Encoder(EncodingJobInfo state, EncodingOptions encodingOptions)
+        {
+            return GetH264OrH265Encoder("libx265", "hevc", state, encodingOptions);
+        }
+
+        private string GetH264OrH265Encoder(string defaultEncoder, string hwEncoder, EncodingJobInfo state, EncodingOptions encodingOptions)
+        {
             // Only use alternative encoders for video files.
             // When using concat with folder rips, if the mfx session fails to initialize, ffmpeg will be stuck retrying and will not exit gracefully
             // Since transcoding of folder rips is expiremental anyway, it's not worth adding additional variables such as this.
```

Comment:
jellyfin fails to user Hardware accleration when trying to stream a BluRay Folder. THis needs to be fixed here.